src/services/user.service.js

```diff
@@ -0,0 +1,45 @@
+const { User } = require('../models/User.model.js');
+
+const getUsers = async () => {
+  const users = await User.findAll();
+
+  return users;
+};
+
+const createUser = (name) => {
+  return User.create({ name });
+};
```

Comment:
The `createUser` function should be marked as `async` to handle potential errors from the `User.create` call. This will allow you to use `try-catch` blocks to manage any database-related errors gracefully.